src/redux/preview.js

```diff
@@ -318,6 +320,7 @@ module.exports.getProjectInfo = (id, token) => (dispatch => {
         if (typeof body === 'undefined') {
             dispatch(module.exports.setFetchStatus('project', module.exports.Status.ERROR));
             dispatch(module.exports.setError('No project info'));
+            dispatch(module.exports.setProjectInfo(null));
```

Comment:
Now that you're handling falsy params to `setProjectInfo`, seems like this could be (a tiny bit) cleaner like:

```
        if (body) {
            dispatch(module.exports.setFetchStatus('project', module.exports.Status.FETCHED));
        } else {
            dispatch(module.exports.setFetchStatus('project', module.exports.Status.ERROR));
            dispatch(module.exports.setError('No project info'));
        } 

        dispatch(module.exports.setProjectInfo(body));
```

Reply:
Just to be clear, is the `body` truthy or falsy when `statusCode === 404`? I'm still wondering if you can simplify the code per my suggestion above!

Reply:
Body seems to be truthy in that case